guides/doc-Release_Notes/topics/foreman-contributors.adoc

```diff
@@ -1,5 +1,4 @@
 We'd like to thank the following people who contributed to the Foreman {{page.version}} release:
 
-Adam Ruzicka, Alex Fisher, Amit Upadhye, Anthony Somerset, Bastian Schmidt, Bernhard Suttner, Chris Roberts, Christian Ruppert, Dirk Götz, Dirk Heinrichs, Dyrkon, Eric D. Helms, Evgeni Golov, Ewoud Kohl van Wijngaarden, Frank Adaemmer, Ian Ballou, Jeremy Lenz, Jonas Trüstedt, Kamil Szubrycht, Karolina Malyjurkova, Leos Stejskal, Lior Keren, Lucy Fu, Lukáš Zapletal, Manuel Laug, Marcel Kühlhorn, Marek Hulán, Maria Agaphontzev, Matt Darcy, Matěj Mudra, Nofar Alfassi, Oleh Fedorenko, Partha Aji, Pat Riehecky, Pavel Moravec, Peter Koprda, Quirin Pamp, Ron Lavi, Samir Jha, Sayan Das, Shim Shtein, William Clark, benjamin-robertson, kobybr, timdeluxe, Štefan Németh
-
+Adam Hosek, Adam Ruzicka, Alex Fisher, Alexey Masolov, Andrew Teixeira, Antonin Dvorak, Archana Kumari, Areyus, Bastian Schmidt, Bernhard Suttner, Damon Clinkscales, Daniel Alley, Dirk Götz, Eric Cook, Eric D. Helms, Erik Berg, Evgeni Golov, Ewoud Kohl van Wijngaarden, Girija Soni, Gordon Bleux, Griffin Sullivan, Ian Ballou, Jan Klare, Jerome Charaoui, Joshua Hoblitt, Leos Stejskal, Lior Keren, Lucy Fu, Marek Hulán, Maria Agaphontzev, Markus Bucher, Maximilian Kolb, Nagoor Shaik, Nofar Alfassi, Oleh Fedorenko, Pat Riehecky, Peter Bittner, Peter Koprda, Romain Tartière, Ron Lavi, Samir Jha, Sebastian Bublitz, Tim Meusel, Trey Dockendorf, Vijaykumar Sawant, William Clark, chr1s692, jc
```

Comment:
Somehow I no longer see Anthony Somerset here.

Recently @wbclark pointed out that you can put everyone on their own line and in the rendered document it ends up as a single sentence. I think the best approach is to replace it first, commit that and then replace the whole set. The line by line comparison should make it very easy to see.

So use `sed -i 's/, /,\n/g' guides/doc-Release_Notes/topics/foreman-contributors.adoc`.

I've also opened https://github.com/theforeman/theforeman.org/pull/2112 to automate the replacement of names.

Reply:
ok I placed them on their own lines. Was I supposed to add Anthony back?

Reply:
I don't have enough context to explicitly know of Anthony was, but I would assume that anyone present as a contributor in 3.7.0 should also be a contributor in 3.7.1